src/main/java/seedu/dietmanager/model/RecipeManager.java

```diff
@@ -8,7 +8,7 @@
 public class RecipeManager {
     private ArrayList<DailyFoodRecord> recipe;
     private static RecipeManager theOnlyOne = null;
-    private static final int MAX_FOOD_TYPES = 4;
+    private static final int MAX_FOOD_TYPES = 3;
```

Comment:
Maybe we don't restrict the user to the number of food types and let them choose for themselves. Maybe a hard cap can be like 5 or 6 instead?